requirements-dev-frozen.txt

```diff
@@ -87,6 +87,7 @@ iniconfig==1.1.1
 iso3166==2.1.1
 isodate==0.6.1
 isort==5.10.1
+jinja2==3.1.6
```

Comment:
We also have django-templates available. That module has very similar syntax to jinja, and you'd be able to use all of our existing template helpers & filters.